smelli/data/yaml/observables_n.yaml

```diff
@@ -0,0 +1,15 @@
+- name: tau_n
+  me_E: 0.655
+- name: Atilde_n
+  me_E: 0.559
+- name: Atilde_n
```

Comment:
@DavidMStraub there two entries for `Atilde_n`. Which one is the correct one?

Reply:
Are they both in Flavio? Could be just two experiments with different kinematics.

Reply:
Ah, yes that's the reason. UCNA has `me_E=0.586` and PERKO has `me_E=0.559`.